### PR TITLE
Add some methods and create unit tests for some classes

### PR DESCRIPTION
**Pull request by danielaSantaCruz**
- DeliveryPipe
- FilterAndReceptacle
- IndicatorLight
- StartButton
- PlateSensor
- Pot

Fix some variable names

### CODE REVIEW

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.prog101CoffeeMaker.coffeeMaker.testPieces;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class StartButtonTest {
+
+    @Test
+    public void isPressed() {
```
Comment (edsonarios) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java]: You must follow the convention about test names, in all methods

Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java]: Ok boss

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java [hunk 2]
```diff
@@ -8,5 +8,19 @@
 
     @Test
     public void getDeliveryWarmWater() {
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java]: the PR has to follow our convention. you need to change the name of method

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java [hunk 3]
```diff
@@ -8,5 +8,19 @@
...
+        DeliveryPipe deliveryPipe = new DeliveryPipe();
+        boolean expected = true;
+        boolean actual = deliveryPipe.getDeliveryWarmWater();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void setDeliveryWarmWater() {
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java]: the PR has to follow our convention. you need to change the name of method

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    public void turnOn() {
+        IndicatorLight indicatorLight = new IndicatorLight();
+        boolean expected = true;
+        indicatorLight.turnOn();
+        boolean actual = indicatorLight.getLightState();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void turnOff() {
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java]: the PR has to follow our convention. you need to change the name of method

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        IndicatorLight indicatorLight = new IndicatorLight();
+        boolean expected = false;
+        indicatorLight.turnOff();
+        boolean actual = indicatorLight.getLightState();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getLight() {
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java]: the PR has to follow our convention. you need to change the name of method

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        IndicatorLight indicatorLight = new IndicatorLight();
+        boolean expected = false;
+        boolean actual = indicatorLight.getLightState();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void setLight() {
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java]: the PR has to follow our convention. you need to change the name of method

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    @Test
+    public void getButtonPressed() {
+        StartButton button = new StartButton();
+        boolean expected = false;
+        boolean actual = button.getButtonPressed();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void setButtonState() {
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java]: the PR has to follow our convention. you need to change the name of method

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java [hunk 8]
```diff
@@ -0,0 +1,44 @@
+package org.fundacionjala.prog101CoffeeMaker.coffeeMaker.testPieces;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class IndicatorLightTest {
+
+    @Test
+    public void turnOn() {
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java]: the PR has to follow our convention. you need to change the name of method

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPot/PotTest.java [hunk 9]
```diff
@@ -7,22 +7,46 @@
...
+        Pot pot = new Pot();
+        boolean newState = true;
+        boolean expected = true;
+        pot.setIsInPlace(newState);
+        boolean actual = pot.getIsInPlace();
+        assertEquals(expected, actual);
     }
 
     @Test
-    public void getAmountOfCups() {
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPot/PotTest.java:22]: the PR has to follow our convention. you need to change the name of method

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+    @Test
+    public void isPressed() {
+        StartButton button = new StartButton();
+        boolean expected = true;
+        boolean actual = button.isPressed();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getButtonPressed() {
```
Comment (cristianfchq) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java]: the PR has to follow our convention. you need to change the name of method

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java (hunk 3 above)
Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java]: Okay thanks for the observation

Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java]: Already done

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java (hunk 5 above)
Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java]: Okay thanks for the observation
Already done

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java (hunk 6 above)
Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java]: Okay thanks for the observation
Already done

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java (hunk 2 above)
Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java]: Okay thanks for the observation

Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java]: Already done

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java (hunk 4 above)
Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java]: Okay thanks for the observation
Already done

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java (hunk 7 above)
Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java]: Okay thanks for the observation

Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java]: Already done

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java (hunk 8 above)
Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/IndicatorLightTest.java]: Okay thanks for the observation
Already done

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java (hunk 10 above)
Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java]: Okay thanks for the observation

Comment (danielaSantaCruz) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/StartButtonTest.java]: done

**Review by cristianfchq — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]

**Pull request merged**